solidity/migrations/2_deploy_contracts.js

```diff
@@ -68,4 +83,20 @@ module.exports = async function (deployer) {
     BondedECDSAKeepVendorImplV1.address,
     implInitializeCallData
   )
+
+  // ETH-only Staking
+  await deployer.deploy(FullyBackedBonding, RegistryAddress)
+
+  await deployer.deploy(FullyBackedECDSAKeep)
+
+  await deployer.deploy(
+    FullyBackedECDSAKeepFactory,
+    FullyBackedECDSAKeep.address,
+    FullyBackedSortitionPoolFactory.address,
+    FullyBackedBonding.address,
+    RandomBeaconAddress
+  )
+
+  // ETH-only backed bonding
+  await deployer.deploy(EthBonding, RegistryAddress)
```

Comment:
I don't think we need it, `FullyBackedBonding` is already defined, I think it's a result of some wrong merge conflict resolution.